arcaflow_plugin_horreum_client/horreum_client_plugin.py

```diff
@@ -61,6 +66,11 @@ def horreum_client(
             verify=params.tls_verify,
         )
 
+    except requests.exceptions.SSLError as e:
+        return "error", ErrorOutput(
+            f"Unable to establish SSL connection with the Horreum server: {e}"
+        )
+
     except (requests.ConnectionError, requests.HTTPError, requests.Timeout):
         return "error", ErrorOutput(
             f"Error communicating with the Horreum server: {auth_return.text}"
```

Comment:
We should have noticed this on the previous round: but you're *not* using `raise_for_status` here so these exceptions, so any exception here was raised by `requests.post` and there will be no `auth_return` Response object to dereference. You should use `{e}` instead...

Reply:
I had the same thought when I looked at it again after inserting the SSL exception capture (which was an uncaught exception in a workflow run). Updated.